calico.yaml

```diff
@@ -0,0 +1,371 @@
+package:
+  name: calico
+  version: 3.25.1
+  epoch: 3
```

Comment:
FYI there's 3.26.0 now.  I just fixed up the automation and merged that for `calicoctl`, but we should update things here as well when you get a chance.